### PR TITLE
add AD search task permission to the list

### PR DESCRIPTION
**Pull request by ylwu-amzn**
*Issue #, if available:*

## Description of changes:
Adding AD search task permission to the list.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: public/apps/configuration/constants.tsx [hunk 1]
```diff
@@ -43,6 +43,7 @@ export const CLUSTER_PERMISSIONS: string[] = [
   'cluster:admin/opendistro/ad/detector/write',
   'cluster:admin/opendistro/ad/detectors/get',
   'cluster:admin/opendistro/ad/result/search',
+  'cluster:admin/opendistro/ad/tasks/search',
```
Comment (saratvemulapalli) [inline public/apps/configuration/constants.tsx:46]: can we add preview and search info APIs as well?
`cluster:admin/opendistro/ad/detector/preview`
`cluster:admin/opendistro/ad/detector/info`

Comment (ylwu-amzn) [reply public/apps/configuration/constants.tsx:46]: Sure, will add it. Send out PR earlier to check if build action can pass. Quite slow to setup local dev environment.

Comment (saratvemulapalli) [reply public/apps/configuration/constants.tsx:46]: Sure thanks @ylwu-amzn !

File: public/apps/configuration/constants.tsx [hunk 2]
```diff
@@ -43,6 +43,9 @@ export const CLUSTER_PERMISSIONS: string[] = [
...
+  'cluster:admin/opendistro/ad/detector/preview',
```
Comment (saratvemulapalli) [inline public/apps/configuration/constants.tsx:47]: Thanks for the change. Just a minor nit: I believe the Security plugin team wants them to be sorted lexicographically.

**Review by saratvemulapalli — APPROVED** [review]

File: public/apps/configuration/constants.tsx (hunk 2 above)
Comment (ylwu-amzn) [reply public/apps/configuration/constants.tsx:47]: Got it, that will look better. Will change

**Review by saratvemulapalli — APPROVED** [review]

**Review by yuxitang-amzn — APPROVED** [review]
Could you add the AD related PR to description? thanks.

**Pull request merged**